README.md

```diff
@@ -1,2 +1,12 @@
-# ruby_homework
+# Ruby homework
 Used for ruby course homework
+
+Following [ruby course](https://github.com/monorkin/learn.rb)
+
+## My favorite programming languages
+* *C*
+* **everything else comes after**
+
+[German shepard](https://previews.123rf.com/images/isselee/isselee1406/isselee140600100/28887721-german-shepherd-dog-puppy-3-months-old-.jpg)
```

Comment:
It's spelled shepherd, not shepard :)
Add `!` here to show the image and you are done with the assignment.